ui-packages/packages/trusty/it-tests/e2e/TrafficViolation.ts

```diff
@@ -0,0 +1,183 @@
+ */
+let reqId;
+let auditDetailsUrl;
+
+describe('Traffic Violation', () => {
+  before(() => {
+    cy.request({
+      method: 'POST',
+      url: 'http://localhost:8080/Traffic Violation',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: {
+        Driver: { State: 'aa', City: 'bb', Age: '25', Points: '13' },
+        Violation: { Type: 'speed', 'Actual Speed': '105', 'Speed Limit': '80' }
+      }
+    }).then(response => {
+      // response.body is automatically serialized into JSON
+      expect(response.body).to.not.be.null;
+      expect(response.body).to.have.property(
+        'Should the driver be suspended?',
+        'No'
+      );
+      expect(response.body).to.have.property('Fine');
+      expect(response.body.Fine).to.have.property('Points', 3);
+      expect(response.body.Fine).to.have.property('Amount', 500);
+      expect(response.headers).to.have.property('x-kogito-execution-id');
+      reqId = response.headers['x-kogito-execution-id'];
+      auditDetailsUrl = `/audit/decision/${reqId}/outcomes`;
+    });
+  });
+
+  it('open Audit Details', () => {
+    cy.visit('/');
+    cy.ouiaId('refresh-button').click();
+    cy.ouiaId(reqId).click();
+    cy.url().should('contains', auditDetailsUrl);
+  });
+
+  describe('verify decision results', () => {
+    beforeEach(() => {
+      cy.visit(auditDetailsUrl);
+    });
+
+    it('Audit Details header', () => {
+      const title = 'Execution #' + String(reqId).substring(0, 8);
+
+      cy.ouiaType('PF4/Breadcrumb')
+        .should('exist')
+        .within($nav => {
+          cy.wrap($nav)
+            .find('li>a')
+            .should('have.length', 3)
+            .within($items => {
+              expect($items.eq(0)).have.text('Audit Investigation');
+              expect($items.eq(1)).have.text(title);
+              expect($items.eq(2)).have.text('Outcomes');
+            });
+        });
+      cy.ouiaId('execution-title')
+        .should('be.visible')
+        .should('contain', title);
+      cy.ouiaId('execution-status')
+        .should('be.visible')
+        .should('contain', 'Completed');
+      cy.ouiaId('nav-audit-detail')
+        .should('exist')
+        .within($nav => {
+          cy.wrap($nav)
+            .find('li>a')
+            .should('have.length', 4)
+            .within($items => {
+              expect($items.eq(0)).have.text('Outcomes');
+              expect($items.eq(0)).have.class('pf-m-current');
+              expect($items.eq(1)).have.text('Outcomes Details');
+              expect($items.eq(2)).have.text('Input Data');
+              expect($items.eq(3)).have.text('Model Lookup');
+            });
+        });
+      cy.get('section.outcomes').should('be.visible');
+    });
+
+    it('Outcomes', () => {
+      cy.get(
+        'div.outcome__property:contains(Points)>div.outcome__property__value'
+      ).should($value => {
+        expect($value).to.have.text('3');
+      });
```

Comment:
I think my previous comment is still valid.
```suggestion
      ).should('have.text', '3');
```